src/utils/explorerClient.ts

```diff
@@ -0,0 +1,45 @@
+/* eslint-disable import/prefer-default-export */
+import * as Logger from './logger'
+
+const makeRequest = async (url: string, opts: RequestInit) => {
+  Logger.breadcrumb(`request: ${opts.method} ${url}`)
+  try {
+    const route = ['https://explorer.helium.com/api', url].join('/')
```

Comment:
Should we add this to the .env?

Reply:
we could it just seemed like extra work to set it up in appcenter for something that wasn't private. I'll just leave it for now